core/vm/jump_table.go

```diff
@@ -47,13 +47,32 @@ type operation struct {
 	// jumps indicates whether operation made a jump. This prevents the program
 	// counter from further incrementing.
 	jumps bool
+	// writes determines whether this a state modifying operation
+	writes bool
 	// valid is used to check whether the retrieved operation is valid and known
 	valid bool
+	// reverts determined whether the operation reverts state
+	reverts bool
 }
 
-var defaultJumpTable = NewJumpTable()
+var (
+	baseInstructionSet      = NewBaseInstructionSet()
```

Comment:
Could we call this `frontierInstructionSet` and `NewFrontierInstructionSet`? We did the same for Frontier/Homesteaddifficulty calculations and also for signers too afaik. Would make the code more uniform.